arangod/Cluster/Maintenance.cpp

```diff
@@ -1078,13 +1086,25 @@ arangodb::Result arangodb::maintenance::phaseTwo (
 
     // maintenace actions
     report.add(VPackValue("actions"));
-    { VPackObjectBuilder agency(&report);
+    { VPackArrayBuilder agency(&report);
       try {
         std::vector<ActionDescription> actions;
         result = syncReplicatedShardsWithLeaders(
-          plan, cur, local, serverId, actions);
+          plan, cur, local, serverId, feature, actions);
 
-        for (auto const& action : actions) {
+        for (auto& action : actions) {
+          LOG_TOPIC(DEBUG, Logger::MAINTENANCE)
```

Comment:
Should LOG_TOPIC occur after potential add of shardVersion?  Otherwise how will logs help if there is an error with that parameter?

Reply:
Good point